src/sonic_ax_impl/mibs/ietf/rfc4292.py

```diff
@@ -79,8 +79,8 @@ def update_data(self):
                 if multi_asic.is_port_channel_internal(ifn, db_conn.namespace):
                     continue
                 if (ifn in port_table and
-                   multi_asic.ROLE in port_table[ifn] and
-                   port_table[ifn][multi_asic.ROLE] == multi_asic.INTERNAL_PORT): 
+                   multi_asic.PORT_ROLE in port_table[ifn] and
+                   port_table[ifn][multi_asic.PORT_ROLE] == multi_asic.INTERNAL_PORT): 
```

Comment:
If `if` statement is line breaked, you may add more indentation to them, so the branch code block is easier to find. There is a PEP.

Reply:
Updated as per comment.